src/shared/js/TestHelper.js

```diff
@@ -0,0 +1,55 @@
+(function() {
```

Comment:
Since this is only related to specific tests, we should put this in the tests/ folder somewhere and <script> include it in the tests that need it. Instead of building it into strand lib.